stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/MessageListView.kt

```diff
@@ -282,15 +286,24 @@ public class MessageListView : ConstraintLayout {
 
     private val DEFAULT_ATTACHMENT_CLICK_LISTENER =
         AttachmentClickListener { message, attachment ->
-            galleryImageAttachmentDestination.apply {
-                this.message = message
-                this.attachment = attachment
-            }
+            val attachmentGalleryItems = message.attachments
+                .filter { it.type == ModelType.attach_image && !it.imageUrl.isNullOrEmpty() }
```

Comment:
`it.imageUrl.isNotEmpty() == true` looks easier to read for me. Of course, this is subjective, do as you prefer.

Reply:
I prefer creating extension function like `someText.hasText()` as such checks are common in our codebase. `it.imageUrl.isNotEmpty() == true` looks a bit weird to me.